index.html

```diff
@@ -91,82 +91,77 @@ <h2>My Recent Works
         <div class="portfolio-item">
```

Comment:
[OPTIONAL] As the main goal of this project kindly remove all the static content for in the middle and generate them dynamically with JavaScript. Check this [diff](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) to understand where you should remove.